modules/nw-anp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,150 @@
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-anp-audit-logging_{context}"]
+= AdminNetworkPolicy audit logging
+
+Audit logging is enabled per `AdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `AdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert", "pass" : "warning" }'
+  name: anp-tenant-log
+spec:
+  priority: 5
+  subject:
+    namespaces:
+      matchLabels:
+        tenant: backend-storage # Selects all pods owned by storage tenant.
+  ingress:
+    - name: "allow-all-ingress-product-development-and-customer" # Product development and customer tenant ingress to backend storage.
+      action: "Allow"
+      from:
+      - pods:
+          namespaceSelector:
+            matchExpressions:
+            - key: tenant
+              operator: In
+              values:
+              - product-development
+              - customer
+          podSelector: {}
+    - name: "pass-all-ingress-product-security"
+      action: "Pass"
+      from:
+      - namespaces:
+          matchLabels:
+              tenant: product-security
+    - name: "deny-all-ingress" # Ingress to backend from all other pods in the cluster.
+      action: "Deny"
+      from:
+      - namespaces: {}
+  egress:
+    - name: "allow-all-egress-product-development"
+      action: "Allow"
+      to:
+      - pods:
+          namespaceSelector:
+            matchLabels:
+              tenant: product-development
+          podSelector: {}
+    - name: "pass-egress-product-security"
+      action: "Pass"
+      to:
+      - namespaces:
+           matchLabels:
+             tenant: product-security
+    - name: "deny-all-egress" # Egress from backend denied to all other tenants.
+      action: "Deny"
+      to:
+      - namespaces: {}
+----
+====
+
+Logs are generated whenever a specific OVN ACL triggers an event that meets the action criteria set in your logging annotation. For example, an event in which any of the namespaces with the label `tenant: product-development` accesses the namespaces with the label `tenant: backend-storage`, a log is generated.
+
+
+[NOTE]
+====
+ACL logging is limited to 60 characters. If your ANP `name` field is long, the rest of the log will be truncated.
+====
+
+The following is a direction index for the examples log entries that follow:
+
+[cols=".^4,.^6a",options="header"]
+|====
+|Direction|Rule
+
+|Ingress
+|
+Rule0:: Allow from tenant `product-development` and `customer` to tenant `backend-storage`; Ingress0: `Allow`
+Rule1:: Pass from `product-security`to tenant `backend-storage`; Ingress1: `Pass`
+Rule2::	Deny all ingress from all other pods; Ingress2: `Deny`
```

Comment:
```suggestion
Rule2::	Deny ingress from all pods; Ingress2: `Deny`
```

Reply:
why was https://github.com/openshift/openshift-docs/pull/75310#discussion_r1635148554 not fixed?

Reply:
I just missed it.